lib/src/just_the_tooltip.dart

```diff
@@ -191,8 +191,22 @@ class _JustTheTooltipOverlayState extends JustTheTooltipState<OverlayEntry> {
   void _createNewEntries() {
     // The builder on these run twice on hot reload and then again from our
     // didUpdateWidget.
+    Widget? overlayEntry;
+
+    try {
+      overlayEntry = _createEntry();
+    } catch (_) {
+      overlayEntry = null;
+    }
+
+    final _overlayEntry = overlayEntry;
```

Comment:
You could inline this by if you want
```
final _overlayEntry = () {
  try {
      return _createEntry();
    } catch (_) {
      return null;
    }
}();
```

Reply:
You know what, I would, but testing the integration with our main project is a tad time-consuming.